### ISSUE TITLE
After ng update to Angular 12.0.4 app won't build and VS Code states it cannot find '@angular/core'

### ISSUE THREAD

**Issue by CodeLiam** [angular/angular#42535]
I thought maybe this was an issue wit the CLI so I filed an issue there with the errors and logs
https://github.com/angular/angular-cli/issues/21101

But the CLI update for 12.0.4 was not out yet.  I updated Angular Material and Angular CLI in my project form 12.0.3 to 12.0.4 and these errors came up.  The app will not build and also will not serve.  I was not sure if there would be something specific in the CLI tied to Angular 12.0.4 but that was not out when this error occurred. 

I apologize if this error is in the wrong place, but it appears these are only resulting from the Core 12.0.4 being updated from 12.0.3, before the update everything was running smoothly.

On a side note, I had ran the update for Angular Material to 12.0.4 and the app was still running without any issues, it was only after updating the Core that this started to occur.

**Comment by alan-agius4** [comment]
Duplicate of https://github.com/angular/angular-cli/issues/21101
